test/e2e/networking.go

```diff
@@ -66,7 +66,7 @@ var _ = Describe("Networking", func() {
 					{
 						Name:    contName,
 						Image:   "gcr.io/google_containers/busybox",
-						Command: []string{"ping", "-c", "3", "-w", "10", "google.com"},
+						Command: []string{"nc", "-w", "30", "google.com", "80"},
```

Comment:
Also lots of incompat across 'nc' implementations.

On Thu, Jun 25, 2015 at 10:02 AM, Quinton Hoole <notifications@github.com>
wrote:

> In test/e2e/networking.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10340#discussion_r33278371>
> :
>
> > @@ -66,7 +66,7 @@ var _ = Describe("Networking", func() {
> >  					{
> >  						Name:    contName,
> >  						Image:   "gcr.io/google_containers/busybox",
> > -						Command: []string{"ping", "-c", "3", "-w", "10", "google.com"},
> > +						Command: []string{"nc", "-w", "30", "google.com", "80"},
>
> I think that if you use the -z flag, works without a TTY. But I don't feel
> strongly either way. wget or curl are also fine.
>
> -z Specifies that nc should just scan for listening daemons, without
> sending any data to them. It is an error to use this option in
> conjunction with the -l option.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10340/files#r33278371>
> .
>